corehq/apps/locations/tasks.py

```diff
@@ -172,3 +172,11 @@ def update_users_at_locations(domain, location_ids, supply_point_ids, ancestor_i
     # update fixtures for users at ancestor locations
     user_ids = user_ids_at_locations(ancestor_ids)
     update_fixture_status_for_users(user_ids, UserFixtureType.LOCATION)
+
+
+def deactivate_users_at_location(location_id):
+    user_ids = update_users_at_locations([location_id])
```

Comment:
`update_users_at_locations` says it also unassigns users; is that your intent? ...and if it is, why not use the `Location.archive` function above in `deprecate_by`?

Reply:
oh no, i meant to use `user_ids_at_locations`

Reply:
https://github.com/dimagi/commcare-hq/pull/26898/commits/ae6b69fd7fdb4fd30b15bc20e31410d3c2ad125b